pennylane/ops/qutrit/non_parametric_ops.py

```diff
@@ -202,3 +203,200 @@ def adjoint(self):
         op = TClock(wires=self.wires)
         op.inverse = not self.inverse
         return op
+
+
+class TAdd(Operation):
+    r"""TAdd(wires)
+    The 2-qutrit controlled add gate
+
+    The construction of this operator is based on definition 7 from
+    `Yeh et al. (2022) <https://arxiv.org/abs/2204.00552>`_.
+
+    .. math:: TAdd = \begin{bmatrix}
+                        1 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 \\
+                        0 & 1 & 0 & 0 & 0 & 0 & 0 & 0 & 0 \\
+                        0 & 0 & 1 & 0 & 0 & 0 & 0 & 0 & 0 \\
+                        0 & 0 & 0 & 0 & 0 & 1 & 0 & 0 & 0 \\
+                        0 & 0 & 0 & 1 & 0 & 0 & 0 & 0 & 0 \\
+                        0 & 0 & 0 & 0 & 1 & 0 & 0 & 0 & 0 \\
+                        0 & 0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 \\
+                        0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 1 \\
+                        0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 & 0
+                    \end{bmatrix}
+
+    .. note:: The first wire provided corresponds to the **control qutrit**.
+
+    **Details:**
+
+    * Number of wires: 2
+    * Number of parameters: 0
+
+    Args:
+        wires (Sequence[int]): the wires the operation acts on
+    """
+    num_wires = 2
+    """int: Number of wires that the operator acts on."""
+
+    num_params = 0
+    """int: Number of trainable parameters that the operator depends on."""
+
+    @staticmethod
+    def compute_matrix():
+        r"""Representation of the operator as a canonical matrix in the computational basis (static method).
+
+        The canonical matrix is the textbook matrix representation that does not consider wires.
```

Comment:
Maybe it means `wire_order` perhaps?

Reply:
It is the canonical matrix you would write if you were considering how the operation acts on the computational basis states in order, in a mathematical context independent of any devices.

Reply:
Ah thanks, in combination with the comment below I understand what is meant now. I think this could be rewritten, most people reading this will probably not understand this from "it does not consider wires".

Reply:
I do agree but I am going to leave as-is for now, as this is written the same way in every single operation and changing them all would be out of scope. It is something better updated in `operation.py` where `compute_matrix` is initially defined.